versions.props

```diff
@@ -1,30 +1,30 @@
 ch.qos.logback:logback-classic = 1.1.7
 com.ea.agentloader:ea-agent-loader = 1.0.3
-com.fasterxml.jackson.*:jackson-* = 2.9.5
+com.fasterxml.jackson.*:jackson-* = 2.9.7
 com.github.ben-manes.caffeine:caffeine = 2.6.2
-com.google.code.findbugs:jsr305 = 3.0.0
+com.google.code.findbugs:jsr305 = 3.0.2
 com.google.guava:guava = 23.6.1-jre
 com.jcraft:jzlib = 1.1.3
 com.netflix.concurrency-limits:* = 0.1.3
-com.netflix.feign:feign-*= 8.17.0
+com.netflix.feign:feign-*= 8.18.0
 com.palantir.conjure.java.api:* = 2.0.0
-com.palantir.safe-logging:* = 1.3.0
+com.palantir.safe-logging:* = 1.5.1
 com.palantir.tracing:* = 1.2.0
 com.palantir.tritium:tritium-registry = 0.8.0
 com.squareup.okhttp3:* = 3.11.0
-com.squareup.retrofit2:* = 2.3.0
+com.squareup.retrofit2:* = 2.4.0
 io.dropwizard.metrics:metrics-core = 3.2.5
 io.dropwizard:dropwizard-* = 1.3.1
 io.reactivex.rxjava2:rxjava = 2.0.4
-javax.servlet:javax.servlet-api = 3.1.0
+javax.servlet:javax.servlet-api = 4.0.1
```

Comment:
Tiny bit unsure if we should be upgrading this... I think we pin this to jetty internally.

Maybe worth:
```
# dependency-upgrader:OFF
```